polyfills/IntersectionObserver/config.json

```diff
@@ -14,11 +14,15 @@
 		"Array.isArray",
 		"Array.prototype.filter",
 		"Array.prototype.forEach",
+		"Array.prototype.indexOf",
 		"Array.prototype.map",
 		"Array.prototype.some",
 		"Function.prototype.bind"
 	],
 	"docs": "https://github.com/WICG/IntersectionObserver/blob/master/explainer.md",
 	"spec": "http://rawgit.com/WICG/IntersectionObserver/master/index.html",
-	"notes": []
+	"notes": [],
+	"repo": "https://github.com/WICG/IntersectionObserver",
+	"module":"intersection-observer",
+	"paths": ["intersection-observer"]
```

Comment:
As above, I'd group these under a single property, possibly `update`, if we are keeping the local source.